docs/get-started/quick_start.md

```diff
@@ -97,7 +104,7 @@ circuit = compiler.compile(inputset)
 
 ## Generating the keys
 
-You can use the `keygen` method of the `Circuit` class to generate the keys (public and private):
+Use the `keygen` method of the `circuit` class to generate the keys (public and private):
```

Comment:
Capitaliszation: **c**ircuit

Reply:
Circuit please. It's Circuit if we speak about the code (and we do here, because of the `Circuit`). For text, it could be ok to speak about circuit